### ISSUE TITLE
Manage subscriptions

### ISSUE THREAD

**Issue by arvedarved** [michael-lazar/rtv#363]
I tried to subscribe to a subreddit from rtv and couldn't find it. 

Similar unsubscribing seems to be missing. 

Either it is undocumented or not implemented :).

As a workaround opening the reddit in  a browser (to subscribe it) would also be useful

**Comment by michael-lazar** [comment]
You're correct that it's not currently implemented, it was requested a long time ago #193. I think it's worth adding but it hasn't been a high priority for me.
